client/me/purchases/manage-purchase/index.jsx

```diff
@@ -295,26 +326,57 @@ class ManagePurchase extends Component {
 		} );
 	};
 
+	getUpgradeUrl() {
+		const { purchase, siteId } = this.props;
+
+		const isUpgradeableBackupProduct = JETPACK_BACKUP_T1_PRODUCTS.includes(
+			purchase.productSlug ?? purchase.product_slug
+		);
+		const isUpgradeableSecurityPlan = JETPACK_SECURITY_T1_PLANS.includes(
+			purchase.productSlug ?? purchase.product_slug
+		);
+
+		if ( isUpgradeableBackupProduct || isUpgradeableSecurityPlan ) {
+			return `/plans/storage/${ siteId }/`;
```

Comment:
We don't seem to need the trailing slash, do we?

Reply:
Yeah, I think they're unnecessary. I only used/kept them here to stay consistent with existing code 🙂